source/includes/quick-start/pojo-query-output.rst

```diff
@@ -1,5 +1,5 @@
 When you run the ``QuickStartPojoExample`` class, it should output the details of the
-movie from the sample dataset which should look something like this:
+movie from the sample dataset which resemble the following:
```

Comment:
[that/which](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-that:~:text=Use%20which%20to%20introduce%20a%20nonrestrictive%20clause%2C%20and%20use%20a%20comma.)

```suggestion
movie from the sample dataset, which resemble the following:
```